CDCgov/prime-data-hub
# 334 - Generate HTML schema docs alongside markup

This PR adds the generation of HTML schema documentation alongside the markup version.  

To test:

1. Run `prime generate-docs` and verify that the markup version of the schema docs are generated.
2. Run `prime generate-docs --generate-html --no-generate-markup` and verify that only the HTML version of the schema docs are generated.

## To review:
The HTML files were generated by the tool.  The real code changes are in the pom.xml and DocumentationFactory.kt.

## Changes
- Use third party library to convert markup to HTML.

## Checklist

### Testing
- [ ] Github build is successful

### Security

### Process

## Security
*List potential security threats and mitigations in the PR*

## Fixes
- https://github.com/CDCgov/prime-data-hub/issues/334
